programs/server/play.html

```diff
@@ -361,23 +361,22 @@
 
     function postImpl(posted_request_num, query)
     {
-        /// TODO: Check if URL already contains query string (append parameters).
+        const user = document.getElementById('user').value;
+        const password = document.getElementById('password').value;
 
-        let user = document.getElementById('user').value;
-        let password = document.getElementById('password').value;
+        const server_address = document.getElementById('url').value;
 
-        let server_address = document.getElementById('url').value;
-
-        let url = server_address +
+        const url = server_address +
+            (~server_address.indexOf('?') ? '&' : '?') + 
```

Comment:
Or even https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes ?
```suggestion
            (server_address.indexOf('?') >= 0 ? '&' : '?') + 
```

Reply:
Whats wrong with `~server_address.indexOf('?')`?

Reply:
It's more difficult to understand, you have to keep in mind bitwise operation instead of comparison. But seems that it's common case in js to use it with `indexOf`, so, I suppose we can leave it